config/gazebo9_ubuntu_bionic.ppa.yaml

```diff
@@ -0,0 +1,8 @@
+# include the fix for server_fixture while Ubuntu merges the patch:
+# https://bugs.launchpad.net/ubuntu/+source/gazebo/+bug/1773811
+name: gazebo9_import_ubuntu_bionic
+method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
+suites: [bionic]
+component: main
+architectures: [amd64, i386, armhf, arm64, source]
```

Comment:
As we currently don't ship i386 I would not add it here tpo match the other gazebo9 import rules